### ISSUE TITLE
Crash on startup - Linux Mint 17 with GLib

### ISSUE THREAD

**Issue by Cervator** [MovingBlocks/Terasology#1810]
Report from IRC, same user (Froward) and system as https://github.com/MovingBlocks/DestinationSol/issues/40

Crashed on game startup, seems to be an error in GLib, specific to Linux systems. This is on a laptop with two GPUs.

Game log: http://pastebin.com/yjEXueE6

Snippet: 

`(process:18611): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed`

System details: http://pastebin.com/CZRxh8Jm

Snippet:

```
00:02.0 VGA compatible controller: Intel Corporation Mobile 4 Series Chipset Integrated Graphics Controller (rev 07)
...
01:00.0 VGA compatible controller: Advanced Micro Devices, Inc. [AMD/ATI] RV620/M82 [Mobility Radeon HD 3450/3470]
```

I found the same error related to a Firefox bug - some links
- http://ubuntuforums.org/showthread.php?t=2175102
- https://bugs.launchpad.net/ubuntu/+source/firefox/+bug/1160569
- http://ubuntuforums.org/showthread.php?t=2238827
- https://bugzilla.mozilla.org/show_bug.cgi?id=8331170

I'm not quite sure what we can do here since it seems specific to the OS, distro, GLib, or maybe video card drivers? One of the links got into the proprietary vs. open source GPU device drivers we've hit on occasion in the past.

May also be difficult to troubleshoot when system specific like this.

@emanuele3d any idea if this could be related to initializing something graphics related or nah?

Edit: It is in the log but to highlight this is for stable release 55, so unrelated to recent work

Edit2: May also want to follow up with http://forum.terasology.org/threads/failed-install.1381/ if progress is made
##

**Comment by Cervator** [comment]
It looks like this *may* be the same issue encountered in Destination Sol just spiraling into a weird error. Froward reports that using the same patched LWJGL 2.9.2 that fixed DestSol also fixed Terasology.

The fix in question, https://github.com/LWJGL/lwjgl/commit/0b83683ed155f089af61b2f4497cfda8181e333a, actually got merged to LWJGL master. However, I don't see it released in a 2.9.4 anywhere. It may exist unofficially / in a nightly build.

We could likewise build it ourselves or use a nightly, then use that version for both Terasology and Destination Sol to fix both issues (its use is transitive though, so we'd need to override it there). 

I remember there was something about the applet we were trying to fix as well that didn't get entirely fixed in 2.9.3. If it is a simple fix we can do ourselves and we're building it anyway ... might be worth a shot.

Likelihood of more official LWJGL 2.x.x releases may be low as it is legacy now with LWJGL 3 out.

**Comment by skaldarnar** [comment]
Closing as outdated.
